sites/partners/page_content/locale_overrides/general.json

```diff
@@ -238,7 +244,7 @@
   "listings.details.listingData": "Listing data",
   "listings.details.updatedDate": "Date updated",
   "listings.details.you": "You",
-  "listings.developer": "Housing developer",
+  "listings.developer": "Property Management Account",
```

Comment:
issue: I believe this name change should only be done for "Non-regulated". For "Regulated" it should remain "Housing developer"